src/Symfony/Component/OpenApi/Builder/OpenApiBuilderInterface.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OpenApi\Builder;
+
+use Symfony\Component\OpenApi\Configurator\CallbackRequestConfigurator;
+use Symfony\Component\OpenApi\Configurator\ComponentsConfigurator;
+use Symfony\Component\OpenApi\Configurator\EncodingConfigurator;
+use Symfony\Component\OpenApi\Configurator\ExampleConfigurator;
+use Symfony\Component\OpenApi\Configurator\InfoConfigurator;
+use Symfony\Component\OpenApi\Configurator\LinkConfigurator;
+use Symfony\Component\OpenApi\Configurator\MediaTypeConfigurator;
+use Symfony\Component\OpenApi\Configurator\OperationConfigurator;
+use Symfony\Component\OpenApi\Configurator\ParameterConfigurator;
+use Symfony\Component\OpenApi\Configurator\PathItemConfigurator;
+use Symfony\Component\OpenApi\Configurator\QueryParameterConfigurator;
+use Symfony\Component\OpenApi\Configurator\QueryParametersConfigurator;
+use Symfony\Component\OpenApi\Configurator\ReferenceConfigurator;
+use Symfony\Component\OpenApi\Configurator\RequestBodyConfigurator;
+use Symfony\Component\OpenApi\Configurator\ResponseConfigurator;
+use Symfony\Component\OpenApi\Configurator\ResponsesConfigurator;
+use Symfony\Component\OpenApi\Configurator\SchemaConfigurator;
+use Symfony\Component\OpenApi\Configurator\SecuritySchemeConfigurator;
+use Symfony\Component\OpenApi\Configurator\ServerConfigurator;
+use Symfony\Component\OpenApi\Configurator\TagConfigurator;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ * @author Selency Team <tech@selency.fr>
+ */
+interface OpenApiBuilderInterface
```

Comment:
Should this be an interface (for which we cannot add methods outside major versions) ? Or just having the OpenApiBuilder allowing us to add more methods if new versions of OpenApi add new features ?